src/clientprocess/from_static_lint.jl

```diff
@@ -72,18 +72,18 @@ end
 function load_module(m, pkg, depot, out)
     out.doc = string(Docs.doc(m))
     out.exported = Set{String}(string.(names(m)))
-    if haskey(depot["manifest"], first(pkg))
-        for pkg1 in depot["manifest"][first(pkg)]
-            if pkg1["uuid"] == last(pkg)
+    if haskey(depot["manifest"], pkg_name(pkg))
```

Comment:
I'm 90% sure that the dict in `depot["manifest"]` is uuid->name.

So I think we also need to make sure that when we import that dict, that we convert the uuids on julia 1.1. to strings in that dict, otherwise the key lookup won't work. I think that probably needs to be done [here](https://github.com/JuliaEditorSupport/SymbolServer.jl/blob/master/src/clientprocess/clientprocess_main.jl#L8).

I think that was the general problem when I looked last: we have a bunch of dicts where the key type is string on julia 1.0 but `UUID` on julia 1.1, because we just get those dicts directly from Pkg. I think we need to convert all those dicts to string->whatever for it to work with both julia 1.0 and 1.1